ocs_ci/deployment/flexy.py

```diff
@@ -333,7 +348,17 @@ def flexy_post_processing(self):
                 f"Symlinking {flexy_cluster_path, abs_cluster_path}"
             )
             os.symlink(flexy_cluster_path, abs_cluster_path)
-            run_cmd(secret_cmd)
+
+        # Apply pull secrets on ocp cluster
+        auth_file = 'flexy/workdir/install-dir/auth/kubeconfig'
+        secret_cmd = (
+            f"oc set data secret/pull-secret "
+            f"--kubeconfig {self.flexy_host_dir}/{auth_file} "
+            f"-n {constants.OPENSHIFT_CONFIG_NAMESPACE} "
+            f"--from-file=.dockerconfigjson={constants.DATA_DIR}/pull-secret"
+        )
+        run_cmd(secret_cmd)
```

Comment:
Don't we need to wait after reload global pull secret with function like this:
https://github.com/red-hat-storage/ocs-ci/blob/e876d7d8c15158ff87d145817350b6cbca63546e/ocs_ci/deployment/deployment.py#L1001

Reply:
Done, depends on https://github.com/red-hat-storage/ocs-ci/pull/3146.